README.md

```diff
@@ -12,7 +12,8 @@
 [The Github repository](netbox-docker-github) houses the components needed to build Netbox as a Docker container.
```

Comment:
This comment should serve as a reminder to (manually, unfortunately) update the README on Docker Hub before merging the project.